src/main.js

```diff
@@ -1,19 +1,19 @@
-import VMapbox from './components/VMapbox.vue';
-import VMapboxLayer from './components/VMapboxLayer.vue';
-import VMapboxSource from './components/VMapboxSource.vue';
-import VMapboxGeolocateControl from './components/VMapboxGeolocateControl.vue';
-import VMapboxNavigationControl from './components/VMapboxNavigationControl.vue';
-import VMapboxGeocoder from './components/VMapboxGeocoder.vue';
+import VMapbox from './components/VMapbox.vue'
```

Comment:
shouldn't the Vue.component be defined in the .js files (inside the components folder), instead of defining them here in the main.js by first importing the javascript?